stable_diffusion.json

```diff
@@ -6698,11 +6698,11 @@
         "baseline": "stable_diffusion_xl",
         "type": "ckpt",
         "inpainting": false,
-        "description": "",
+        "description": "DreamShaper is a general purpose SD model that aims at doing everything well, photos, art, anime, manga. It's designed to go against other general purpose models and pipelines like Midjourney and DALL-E.",
         "version": "3.0",
         "style": "anime",
         "homepage": "https://civitai.com/models/112902",
-        "nsfw": false,
+        "nsfw": true,
```

Comment:
Is it NSFW?  How do you determine it btw?

Reply:
![image](https://github.com/Haidra-Org/AI-Horde-image-model-reference/assets/2823978/4ee0888f-111a-4245-bc82-9fe27ad72af4)

Any model with NSFW training is NSFW.

Reply:
https://civitai.com/models/112902?modelVersionId=354657